PYME/Acquire/SpoolController.py

```diff
@@ -306,8 +317,6 @@ def LaunchAnalysis(self):
                 if self._analysis_launchers.full():
                     self._analysis_launchers.get().join()
                 self._analysis_launchers.put(t)
-            else:
```

Comment:
This has to remain - otherwise the workflow for non-high throughput imaging with the cluster is broken. Current workflow is as follows:

click "Series", protocol launches dh5view when enough frames are in to start analysis, check parameters in dh5view, click "go". 

If your issue is that you are getting loads of dh5view windows popping up when you spool to the cluster without autoanalysis, then change the protocol so that it doesn't launch analysis.